script.js

```diff
@@ -1,52 +1,62 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] **Optional:** You have done an amazing job! 👏👏 You can consider breaking the class into smaller files by modularizing your code using `import`s  and `export`s.